rfcs/0002-ipfs-governance.md

```diff
@@ -0,0 +1,215 @@
+- People _can_ belong to multiple Teams, but should avoid over committing. **People whose time is split across teams do not count toward either Team's 3-person minimum.**
+- Captains (team leads) are responsible for keeping track of whether people are over committed (including keeping track of whether people have overcommitted across multiple Teams)
+
+### Working Groups
+
+- A Working Group can't form until there are multiple Teams that need it to exist
+- Working Groups don't necessarily have any people allocating full-time effort to that group's goals. As a result, Working Groups are likely to push work into the OKRs of Teams, which can more easily allocate people's time.
+- Working Groups occasionally define OKRs, but they don't have to define OKRs every quarter.
+- Working Groups don’t necessarily subdivide when they get bigger. Unlike Teams, which are encouraged to form a tree of sub-teams when they grow bigger than ~5 people, Working Groups are able to convene larger groups of stakeholders.
+
+### The Core Working Group
+
+This whole structure and process relies on a Core Working Group, which acts in service of the other Teams and Working Groups to maintain a healthy, adaptable system for decision making and coordinated planning.
+
+The Core Working Group has three main functions:
+
+1. Set overall direction of the project
+2. Keep the Working Group lifecycle running smoothly
+3. Keep the RFC process running smoothly, delegating the decision making to Working Groups whenever possible
+
+### The Core Working Group is Responsible for
+- Delivering high-level, long-term strategic OKRs
+- Enforcing the lifecycle protocol for Teams and Working Groups: Guides formation of Teams/WGs, supports functioning of Teams & WGs, and dissolves them according to the lifecycle protocol
+- Keep the RFC process moving -- either delegating decision power to whichever Team/WG should make the decision or, when no other group matches the RFC, merging/rejecting the RFC themselves.
+
+### Prerogatives of the Core Working Group
+- Core team can assign/delegate ownership of a product, library, spec, problem domain to a Team or WG, So that those groups can operate with clear mandates and healthy amounts of autonomy.
+- In case of process failure or broken protocols among the teams, the Core Working Group is responsible for addressing the situation.
+- They have the authority to institute short-term measures to address pressing problems/disputes
+- They have the authority to fast-track RFCs that amend broken processes or protocols.
```

Comment:
Maybe we can get a short list of what the Core working group cannot do, or things which are not among their prerogatives.